api-reference/beta/resources/connector.md

```diff
@@ -12,8 +12,8 @@
 
 | Method		   | Return Type	|Description|
 |:---------------|:--------|:----------|
-|[Get connector](../api/connector_get.md) | [connector](connector.md) |Read properties and relationships of connector object.|
-|[List memberOf](../api/connector_list_memberof.md) |[connectorGroup](connectorgroup.md) collection| Get the connectorGroup object associated with the connector.|
+|[Get connectors](../api/connector_get.md) | [connectors](connector.md) |Read properties and relationships of connectors object.|
```

Comment:
OK - little confused...
There should be 3 Methods, I think:
1. Get connector (this is the get on a specific connector item - GET /connectors/{id})
2. List connectors - GET /connectors
3. List memberOf - again this is a collection of connectorGroup (not connectorGroup).